ledger-api/grpc-definitions/com/daml/ledger/api/v1/in_flight_transfers_service.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+import "com/daml/ledger/api/v1/event.proto";
+import "com/daml/ledger/api/v1/transaction_filter.proto";
+import "com/daml/ledger/api/v1/transfer.proto";
+
+option java_outer_classname = "InFlightTransfersServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Allows clients to get in-flight state of transfers from the ledger.
+service InFlightTransfersService {
+
+  // Get the list of in-flight transfers.
+  // If there are no in-flight transfers, the stream returns a single response message with the offset at which the snapshot has been taken.
+  // Clients SHOULD use the offset in the GetInFlightTransfersResponse message to continue streaming updates with the update service.
+  // Clients SHOULD NOT assume that the set of in-flight transfers they receive reflects the state at the ledger end.
+  // Clients SHOULD be aware of limitations of this endpoint.
+  rpc GetInFlightTransfers (GetInFlightTransfersRequest) returns (stream GetInFlightTransfersResponse);
+}
+
+message GetInFlightTransfersRequest {
+  // Must be a valid domain ID
+  // Required
+  string source_domain_id = 1;
```

Comment:
I think you want to get the in-flight transfers for all domains.

Reply:
yes, that is a WIP